console/src/cluster/Dropdown.tsx

```diff
@@ -64,29 +63,28 @@ export const List = (): ReactElement => {
     Text.edit(`text-${key}`);
   };
 
-  const handleLink = (key: string): void => {
-    void navigator.clipboard.writeText(`synnax://cluster/${key}`);
-  };
+  const handleLink = Link.useCopyToClipboard();
 
   const contextMenu = useCallback(
     ({ keys: [key] }: PMenu.ContextMenuMenuProps): ReactElement | null => {
       if (key == null) return <Layout.DefaultContextMenu />;
       const handleSelect = (menuKey: string): void => {
-        if (key == null) return;
         switch (menuKey) {
           case "remove":
             return handleRemove([key]);
           case "connect":
             return handleConnect(key);
           case "disconnect":
             return handleConnect(null);
-          case "link":
-            return handleLink(key);
+          case "link": {
+            const name = allClusters.find((c) => c.key === key)?.name;
+            if (name == undefined) return;
```

Comment:
I almost always used `value == null` for convention. Just my style. This works too though

Reply:
I used `name == undefined` here since that was the actual case I needed to check, but I'll switch to `null` for consistency.